dasharo-security/measured-boot.robot

```diff
@@ -18,25 +18,376 @@ Resource            ../keys.robot
 # - document which setup/teardown keywords to use and what are they doing
 # - go threough them and make sure they are doing what the name suggest (not
 # exactly the case right now)
-Suite Setup         Run Keyword
-...                     Prepare Test Suite
+Suite Setup         Measured Boot Suite Setup
 Suite Teardown      Run Keyword
 ...                     Log Out And Close Connection
 
 
+*** Variables ***
+${PCRS_TO_CHECK}=       [0-9]|14
+
+
 *** Test Cases ***
 MBO001.001 Measured Boot support (Ubuntu 20.04)
     [Documentation]    Check whether Measured Boot is functional and
-    ...    measurements are stored into the TPM.a.
-    Skip If    not ${MEASURED_BOOT_SUPPORT}    MBO001.001 not supported
-    Skip If    not ${TESTS_IN_UBUNTU_SUPPORT}    MBO001.001 not supported
+    ...    measurements are stored into the TPM.
+    Execute Command In Terminal    shopt -s extglob
+    ${hashes}=    Execute Command In Terminal
+    ...    grep . /sys/class/tpm/tpm0/pcr-sha@(1|256)/[0-3]
```

Comment:
A comment that `grep` is used to get file names in the output would be helpful (there is another use at the bottom).

Reply:
Added comment in [f6fdd1e](https://github.com/Dasharo/open-source-firmware-validation/pull/326/commits/f6fdd1eca99d3349aa0be11e35aa2a58307a4141) and added `-H` flag in probably impossible case when there is only one PCR in `pcr-sha` directory